lib/public/components/qcFlags/badRunInAsyncQcMessage.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RunQualities } from '../../domain/enums/RunQualities.js';
+
+export const BAD_RUN_IN_ASYNC_QC_MESSAGE = `Quality of the run was changed to ${RunQualities.BAD} so it is no more subject for QC`;
```

Comment:
Why mentioning the change? I would rather put "Quality of the run is ${RunQualities.Bad} and is therefore not subject to QC flags."

Reply:
because the quality must have been changed, it must has been good (only the it's taken to ML) and then changed to bad

Reply:
Ok I understand